server/index.js

```diff
@@ -1,15 +1,22 @@
 require("dotenv").config();
 const express = require("express");
 const cors = require("cors");
+const sequelize = require("./db");
```

Comment:
I'd rather name it "dbConnector" or something like that. We will probably change the ORM to any other and we had to rename this const.
Btw, why don't you use ES6 import? (import any from "anywhere")?